libs/features/pages/consultancy-page/src/lib/elewa-consultancy-about-one/elewa-consultancy-about-one.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="image-text-banner" [ngClass]="getLayoutClass()">
+  <div class="image-container">
+    <!-- Check if ImageAndText.image is defined -->
+    <img
+      *ngIf="ImageAndText?.image"
+      [src]="ImageAndText.image.imageSrc"
+      [alt]="ImageAndText.image.title"
+      [style.max-width.px]="
+        ImageAndText.image.maxWidth ? ImageAndText.image.maxWidth : undefined
+      "
+    />
+  </div>
+  <div class="text-container">
+    <!-- Use content -->
+    <elewa-website-elewa-text-content-item
+      [content]="content"
+    ></elewa-website-elewa-text-content-item>
+  </div>
+</div>
```

Comment:
As specified, the text and image banner component should be reused

Reply:
@AnnaL001 *resolved